framework/web/AssetManager.php

```diff
@@ -290,6 +299,7 @@ protected function loadDummyBundle($name)
     /**
      * Returns the actual URL for the specified asset.
      * The actual URL is obtained by prepending either [[baseUrl]] or [[AssetManager::baseUrl]] to the given asset path.
+     * [[scheme]] or [[AssetManager::scheme]] is used, if specified.
```

Comment:
If both specified:
1. and asset is missing from AssetManager::assetMap, then bundle scheme will be used.
2. and asset matched in assetMap, then AssetManager::scheme will be used.
So, AssetManager::scheme overwrites AssetBundle::schema, if asset exists in AssetManager::assetMap.

Perhaps, to keep it simple it's better to remove AssetsManager::scheme and leave only AssetBundle::scheme.
What do you think? It will be enough to solve issue.